Description: This takes longer on mips64, so increasee the timeout to make sure
it has time to finish on mips64
Upstream: no

diff --git a/hugolib/pagebundler_test.go b/hugolib/pagebundler_test.go
index fa420a0..34d928a 100644
--- a/hugolib/pagebundler_test.go
+++ b/hugolib/pagebundler_test.go
@@ -90,6 +90,7 @@ func TestPageBundlerSiteRegular(t *testing.T) {
 						})
 
 						cfg.Set("uglyURLs", ugly)
+						cfg.Set("timeout", 120000)
 
 						b := newTestSitesBuilderFromDepsCfg(c, deps.DepsCfg{Logger: loggers.NewErrorLogger(), Fs: fs, Cfg: cfg}).WithNothingAdded()
 

